test/validators/remove.ts

```diff
@@ -23,205 +23,389 @@ describe('Remove Validator Tests', () => {
-    ));
-    firstCluster = register.eventsByName.ValidatorAdded[0].args;
-
-    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1]).removeValidator(
-      helpers.DataGenerator.publicKey(2),
-      firstCluster.operatorIds,
-      firstCluster.cluster
-    ), [GasGroup.REMOVE_VALIDATOR_7]);
+    const cluster = await helpers.registerValidators(
+      1,
+      (minDepositAmount * 2).toString(),
+      [2],
+      helpers.DEFAULT_OPERATOR_IDS[7],
+      helpers.getClusterForValidator(0, 0, 0, 0, true),
+      [GasGroup.REGISTER_VALIDATOR_NEW_STATE_7],
+    );
+    firstCluster = cluster.args;
+
+    await trackGas(
+      ssvNetworkContract
+        .connect(helpers.DB.owners[1])
+        .removeValidator(helpers.DataGenerator.publicKey(2), firstCluster.operatorIds, firstCluster.cluster),
+      [GasGroup.REMOVE_VALIDATOR_7],
+    );
+  });
+
+  it('Bulk remove 10 validator gas limit (7 operators cluster)', async () => {
+    const { args, pks } = await helpers.bulkRegisterValidators(
+      2,
+      10,
+      helpers.DEFAULT_OPERATOR_IDS[7],
+      minDepositAmount,
+      helpers.getClusterForValidator(0, 0, 0, 0, true),
+    );
+
+    await trackGas(
+      ssvNetworkContract
+        .connect(helpers.DB.owners[2])
+        .bulkRemoveValidator(pks, args.operatorIds, args.cluster),
+      [GasGroup.BULK_REMOVE_10_VALIDATOR_7],
+    );
   });
 
   it('Remove validator gas limit (10 operators cluster)', async () => {
-    await helpers.DB.ssvToken.connect(helpers.DB.owners[1]).approve(ssvNetworkContract.address, minDepositAmount * 3);
-
-    const register = await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1]).registerValidator(
-      helpers.DataGenerator.publicKey(2),
-      [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
-      helpers.DataGenerator.shares(10),
-      minDepositAmount * 3,
-      {
-        validatorCount: 0,
-        networkFeeIndex: 0,
-        index: 0,
-        balance: 0,
-        active: true
-      }
-    ));
-    firstCluster = register.eventsByName.ValidatorAdded[0].args;
-
-    await trackGas(ssvNetworkContract.connect(helpers.DB.owners[1]).removeValidator(
-      helpers.DataGenerator.publicKey(2),
-      firstCluster.operatorIds,
-      firstCluster.cluster
-    ), [GasGroup.REMOVE_VALIDATOR_10]);
+    const cluster = await helpers.registerValidators(
+      1,
+      (minDepositAmount * 3).toString(),
+      [2],
+      helpers.DEFAULT_OPERATOR_IDS[10],
+      helpers.getClusterForValidator(0, 0, 0, 0, true),
+      [GasGroup.REGISTER_VALIDATOR_NEW_STATE_10],
+    );
+    firstCluster = cluster.args;
+
+    await trackGas(
+      ssvNetworkContract
+        .connect(helpers.DB.owners[1])
+        .removeValidator(helpers.DataGenerator.publicKey(2), firstCluster.operatorIds, firstCluster.cluster),
+      [GasGroup.REMOVE_VALIDATOR_10],
+    );
+  });
+
+  it('Bulk remove 10 validator gas limit (10 operators cluster)', async () => {
+    const { args, pks } = await helpers.bulkRegisterValidators(
```

Comment:
In my environment, this test case was failed with this error message.
`Error: VM Exception while processing transaction: reverted with custom error 'InsufficientBalance()'`